components/rewards/RewardsFooter.js

```diff
@@ -1,51 +1,77 @@
+import { getCustomersById } from '../../lib/airtable/request';
+import { logErrorToSentry } from '../../lib/logUtils';
 import { Subhead } from '../BaseComponents';
 /**
  * @prop
  * */
 
-function RewardsFooter({ customer, isGuest }) {
-  return (
-    <View
-      style={{
-        width: '100%',
-        flexDirection: 'row',
-        justifyContent: 'space-between',
-        padding: 16,
-      }}>
-      {customer && (
-        <View
-          style={{
-            flexDirection: 'row',
-            alignItems: 'center',
-          }}>
-          <FontAwesome5 name="star" solid size={16} color={Colors.lightest} />
-          <Subhead style={{ paddingLeft: 8 }} color={Colors.lightest}>
-            {isGuest
-              ? 'Learn about Healthy Rewards >'
-              : `${customer.points} points`}
-          </Subhead>
-        </View>
-      )}
-      {customer && !isGuest && (
-        <Subhead color={Colors.lightest}>Your next reward &gt;</Subhead>
-      )}
-    </View>
-  );
-}
-
-RewardsFooter.propTypes = {
-  customer: PropTypes.object,
-  isGuest: PropTypes.bool,
-};
+export default class RewardsFooter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      customer: null,
+      isGuest: false,
+      rewards: 0,
+    };
+  }
 
-RewardsFooter.defaultProps = {
-  customer: null,
-  isGuest: true,
-};
+  async componentDidMount() {
+    const customerId = await AsyncStorage.getItem('customerId');
+    const isGuest = customerId === RecordIds.guestCustomerId;
+    try {
+      const customer = await getCustomersById(customerId);
+      const rewards = customer
+        ? Math.floor(parseInt(customer.points, 10) / rewardPointValue)
+        : 0;
+      this.setState({
+        isGuest,
+        customer,
+        rewards,
+      });
+    } catch (err) {
+      console.error(err);
+      logErrorToSentry({
+        screen: 'RewardsFooter',
+        action: 'componentDidMount',
+        error: err,
+      });
+    }
+  }
 
-export default RewardsFooter;
+  render() {
+    return (
+      <View
+        style={{
+          width: '100%',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          padding: 16,
+        }}>
+        {this.state.customer && (
+          <View
+            style={{
+              flexDirection: 'row',
+              alignItems: 'center',
+            }}>
+            <FontAwesome5 name="star" solid size={16} color={Colors.lightest} />
+            <Subhead style={{ paddingLeft: 8 }} color={Colors.lightest}>
+              {this.state.isGuest
+                ? 'Learn about Healthy Rewards >'
+                : `${this.state.rewards} rewards`}
+            </Subhead>
+          </View>
+        )}
+        {/* &gt; is a code for >, but since we can't use that, we use &gt; */}
+        {this.state.customer && !this.state.isGuest && (
+          <Subhead color={Colors.lightest}>View your rewards &gt;</Subhead>
```

Comment:
This should only show # rewards if they have rewards. It should show points if they don't have any. Check the latest Figma updates!